environments/dev/main.jsonnet

```diff
@@ -34,6 +34,35 @@ local dex = (import '../../components/dex.libsonnet') + {
 };
 
 local obs = (import '../base/observatorium.jsonnet') + {
+  tls_secret: {
+    apiVersion: 'v1',
+    kind: 'Secret',
+    metadata: {
+      name: 'observatorium-api-tls-certs',
+      namespace: obs.config.namespace,
+    },
+    data: {
+      'server.key': importstr '../../tmp/base64_certs/observatorium/server.key',
+      'server.pem': importstr '../../tmp/base64_certs/observatorium/server.pem',
+      'client.pem': importstr '../../tmp/base64_certs/observatorium/client.pem',
+      'client.key': importstr '../../tmp/base64_certs/observatorium/client.key',
+    },
+    stringData: {
+      'ca.pem': importstr '../../tmp/certs/observatorium/ca.pem',
+    },
+    type: 'Opaque',
+  },
+  tls_configmap: {
+    apiVersion: 'v1',
+    kind: 'ConfigMap',
+    metadata: {
+      name: 'observatorium-api-tls-client-ca',
+      namespace: obs.config.namespace,
+    },
+    data: {
+      'ca.pem': importstr '../../tmp/certs/up_client/ca.pem',
```

Comment:
The reason that mTLS is failing is that you have generated two different sets of client certs each with a different CA. The client certs for the up binary were generated using this certificate, so using this CA as the client-ca means up will work. *But* the client certs for the healthchecks were generated using the observatorium/ca.pem cert, so when the API checks the client certificates against the up_client/ca.pem cert, the validation will fail

Reply:
I have tested this as well, and Done per your request. Yet I still get the`TLS handshake error`